CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 ## Unreleased
 * Fixed an issue where logging arguments were not in the standard kebab-case. The new arguments are: **console-log-threshold**, **file-log-threshold**, **log-file-path**.
+* Demisto-SDK will now exit gracefully with an appropriate error message when *git* is not installed.
```

Comment:
please document in `README.md`, that `Git` is required for using the SDK.

Reply:
on it